### DISCUSSION TITLE
Grouped dependencies declined PR is not being ignored on next run

### DISCUSSION THREAD

**Discussion by bemccandless** [renovatebot/renovate#22773] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.105.1

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

When grouping dependencies together such as,
`{
      "groupName": "angular",
      "matchPackagePatterns": [
        "@angular/"
      ]
}`
and declining the PR that was created I would expect a PR to not be created again in the next scheduled run, just like other dependencies. The process is working for regular dependencies, but is failing for grouped ones.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
https://docs.renovatebot.com/key-concepts/pull-requests/#immortal-prs
